src/ray/gcs/gcs_server/gcs_server.h

```diff
@@ -172,6 +174,8 @@ class GcsServer {
   /// Initialize monitor service.
   void InitMonitorServer();
 
+  friend class ray::GcsClientTest;
```

Comment:
Can we use FRIEND_TEST from gtest?

Reply:
I guess the particular method that uses this, `GcsClientTest::StampContext`, is called in 9 different tests, so having declaration and `FRIEND_TEST` for each individual one might be a bit more code clutter